README.md

```diff
@@ -1,2 +1,54 @@
-# admin.orders.api.ch.gov.uk
-Internal Admin web application tool used to examine orders.
+# admin.orders.web.ch.gov.uk
+
+## Summary
+
+* Internal Admin web application tool used to examine orders.
+* Examiners will be able to inspect details of paid orders 
+for further processing.
+
+## System requirements
+
+* [Git](https://git-scm.com/downloads)
+* [NodeJS](https://nodejs.org/en/)
+
+## Architecture
+
+* The admin orders web app has been developed using [ExpressJS](https://expressjs.com/) and [TypeScript](https://www.typescriptlang.org/).
+* The template engine used is [Nunjucks](https://mozilla.github.io/nunjucks/), 
+to follow the standards set by the [GovUK Design System](https://design-system.service.gov.uk/).
+* Unit tests use the [Jest](https://jestjs.io/) testing framework and [Mocha](https://mochajs.org/).
+* Integration tests use [CucumberJS](https://cucumber.io/docs/installation/javascript/), [Gherkin](https://cucumber.io/docs/gherkin/), 
+Jest & [TestContainers](https://github.com/testcontainers/testcontainers-node).
+
+
+## Environment variables
+
+* List of environment variables required to be configured for the application to run.
+
+|Variable                                      |Description                                                  |Example            |
+|----------------------------------------------|-------------------------------------------------------------|-------------------|
+| To be added.                                 | To be added.                                                | To be added.      |    
+
+
+## Building the docker image
+
+    mvn compile jib:dockerBuild -Dimage=169942020521.dkr.ecr.eu-west-1.amazonaws.com/local/admin.orders.web.ch.gov.uk
```

Comment:
This isn't a Java project; let's get devs to build/test/run the project via a Makefile.

Reply:
Resolved in latest commit.